### DISCUSSION TITLE
How to force `next dev` to do Full Reloads instead of Fast Refreshes?

### DISCUSSION THREAD

**Discussion by sevaseva** [vercel/next.js#48015] [Help]
### Summary

I would like my `next dev` server to do a Full Reload any time it thinks a Fast Refresh is sufficient. How can I configure or hack it to do that?

---

Background:
I am iterating on code that's only imported on the client side, i.e. along these lines:

```
import dynamic from "next/dynamic"
const FooComponent = dynamic(() => import('foo'), { ssr: false })
export default function MyComponent...
...
const [isLoaded, setLoaded] = useState(false)
useEffect(() => setLoaded(true), [])
...
   return (
        <>
            {isLoaded && <FooComponent />}
...

```

When I make a change in that `foo.js` or its dependencies, my `next dev` server does a fast refresh that doesn't effectively pull in/rerun my change in my open browser tab. And so I need something else to refresh that tab if I want to see my change in action.

---

Forcing the `next dev` server to always do a Full Reload is one brute force way to solve this problem that would work for me. Any other advice is welcome.

Thanks!

### Additional information

```js
npm next --version
8.19.2
```

**Comment by dieterc** [comment]
Any solution for this?

**Reply by sevaseva** [reply]
`export const dummy = 0` in every file that you want to force full refresh when updated

**Reply by dieterc** [reply]
For me that doesn't work...
My **next** version is **10.1.0**

**Comment by SnowdenWintermute** [comment]
https://nextjs.org/docs/architecture/fast-refresh#tips

If you look at their docs here you can add 
```// @refresh reset```
which helped me because when I was editing some babylon.js files it wasn't refreshing/remounting my components that load the scene. Works great now.
